eng/automation/generate_utils.py

```diff
@@ -414,9 +417,6 @@ def generate_typespec_project(
                     # clear existing generated source code, and regenerate
                     drop_changes(sdk_root)
                     remove_generated_source_code(sdk_folder, f"{group_id}.{service}")
-                    _, current_version = set_or_default_version(sdk_root, group_id, module, version=version)
-                    tsp_cmd.append("--emitter-options")
-                    tsp_cmd.append(f'package-version={current_version}')
                     # regenerate
```

Comment:
This property is originally intended for getting package version during SDK runtime. Now that it's obtained by properties file, we no longer need to set it.

Reply:
Yeah, guess we still need it..

Reply:
added back